app/services/xccdf_report_parser.rb

```diff
@@ -51,21 +58,65 @@ def check_for_external_reports
     # rubocop:disable Style/GuardClause
     if external_report?
       raise ExternalReportError,
-            "No policy found matching benchmark #{benchmark.ref_id} "\
-            "(RHEL-#{benchmark.os_major_version}) and profile "\
-            "#{test_result_profile.ref_id} with host #{@host.name} assigned "\
-            "for account #{@account.account_number}."
+            "No policy found matching benchmark #{benchmark.ref_id}. "\
+            "#{parse_failure_message}"
     end
     # rubocop:enable Style/GuardClause
   end
 
+  def check_for_missing_benchmark
+    # rubocop:disable Style/GuardClause
+    unless benchmark.persisted?
```

Comment:
If this is the only condition, then it is too narrow. It has to bypass it for cases when the benchmark isn't imported yet but it is still on the supported matrix. This bypass has to exist, until all necessary downstream SSGs are imported.

If there would be another upstream vs. downstream version mismatch (backport) then it would not allow to import these benchmarks (unless I've missed something).

Reply:
Hmm, that's a good point. Perhaps we need to import all downstream versions, in addition to any upstream version required. So we'd import RHEL 7 0.1.49 and 0.1.50. The whole point with this change is to only import benchmarks during the ssg import task and never during report parsing.

Reply:
Sorry, I don't understand. We import all the versions listed in the support matrix during the import_ssg job, not just the latest supported versions for each major OS. Which versions are missing assuming we also import upstream versions?

Reply:
Ok, I see what you mean now. There are some versions that don't exist at all upstream. I'll add some logic to keep importing those versions if they don't already exist.

Reply:
Updated